### PR TITLE
[IMP] product_print_category: allow configuration on product.template

### PR DESCRIPTION
**Pull request by remytms**
This commit allow to configure print options on product.template if product variants are not activated.

[coopiteasy tasks](https://gestion.coopiteasy.be/web#id=6483&view_type=form&model=project.task&action=475&active_id=208)

### CODE REVIEW

**Comment by remytms** [comment]
@legalsylvain This PR is a proposal to let module `product_print_category` compatible with database where product variants are not activated.

This is not yet tested from our side. I will let you know when it will be done. :)

File: product_print_category/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2021-Today: Coop IT Easy (<http://coopiteasy.be>)
+# @author: Rémy TAYMANS (<remy@coopiteasy.be>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
```
Comment (legalsylvain) [inline product_print_category/models/product_template.py]: api is unused. (makes travis red)

**Review by legalsylvain — COMMENTED** [review]
LGTM ! Let me know when it is tested from your side.

**Comment by legalsylvain** [comment]
Hello @remytms ! 

1) I fixed trivial travis error here : https://github.com/coopiteasy/grap-odoo-incubator/pull/1 could you merge it ? 

2) Did you tested the PR ? 

thanks for this improvment.

**Comment by vdewulf** [comment]
@legalsylvain and @remytms 
Yes, I tested this PR on our test database and it works perfectly! No need to activate the variants option in order to see the specific fields for the printing on the products.
Thanks to both of you!

**Comment by legalsylvain** [comment]
Remy, could you merge the PR coopiteasy#1  on your  branch ? I can not merged because Travis is red.

thanks

**Comment by legalsylvain** [comment]
MeP du 28/07/2021.

/ocabot merge patch

**Pull request merged**